istio/istio.spec

```diff
@@ -380,13 +380,24 @@ cp %{SOURCE2} ISTIO/src/istio.io/istio/buildinfo
 
 %build
 cd ISTIO
-export GOPATH=$(pwd):%{gopath}
+export GOPROXY=off
+export GOPATH=$(pwd):${GOPATH}
+
+export GOARCH=${GOARCH:-'amd64'}
+ISTIO_OUT=$(pwd)/out/linux_${GOARCH}/release
```

Comment:
I'm not confident that this line does anything since the makefile redefines it:

https://github.com/Maistra/istio/blob/maistra-1.1/Makefile.core.mk#L45...L50
`
# If GOPATH is not set by the env, set it to a sane value
GOPATH ?= $(shell cd ${ISTIO_GO}/../../..; pwd)
export GOPATH

# If GOPATH is made up of several paths, use the first one for our targets in this Makefile
GO_TOP := $(shell echo ${GOPATH} | cut -d ':' -f1)
`

https://github.com/Maistra/istio/blob/maistra-1.1/Makefile.core.mk#L137...L140

`export ISTIO_BIN=$(GO_TOP)/bin
# Using same package structure as pkg/
export OUT_DIR=$(GO_TOP)/out
export ISTIO_OUT:=$(OUT_DIR)/$(GOOS)_$(GOARCH)/$(BUILDTYPE_DIR)`